src/mxbase.jl

```diff
@@ -41,6 +41,12 @@ function get_paths()
             error("The MATLAB path is invalid. Set the MATLAB_HOME evironmental variable to the MATLAB root.")
         end
         default_startcmd = "exec $(Base.shell_escape(default_startcmd)) -nosplash"
+
+        # Disable DISPLAY if asked or not avaible
+        if get(ENV, "DISPLAY", nothing) == nothing ||
+            get(ENV, "MATLAB_NO_DISPLAY", nothing) != nothing
```

Comment:
Not sure if this should really be an environmental variable, as opposed to an argument to `MSession` and `restart_default_msession`.